lib/rules/font-family-no-missing-generic-family-keyword/index.js

```diff
@@ -23,6 +24,12 @@ const messages = ruleMessages(ruleName, {
 const isFamilyNameKeyword = (node) =>
 	!node.quote && keywordSets.fontFamilyKeywords.has(node.value.toLowerCase());
 
+const isLastFontFamilyVariable = (value) => {
+	const lastValue = postcss.list.comma(value).pop();
```

Comment:
[note] `postcss.list.comma()` seems more suitable than `postcss.list.space()` in this case.

See also <https://postcss.org/api/#list>